src/xenia/kernel/xam/content_manager.cc

```diff
@@ -62,24 +60,32 @@ ContentManager::ContentManager(KernelState* kernel_state,
 ContentManager::~ContentManager() = default;
 
 std::filesystem::path ContentManager::ResolvePackageRoot(
-    XContentType content_type, uint32_t title_id) {
+    uint64_t xuid, uint32_t title_id, XContentType content_type) const {
   if (title_id == kCurrentlyRunningTitleId) {
     title_id = kernel_state_->title_id();
   }
+
+  if (content_type == XContentType::kMarketplaceContent) {
```

Comment:
For cases when game is "installed" usually it is booted via: ``XamContentLaunchImageFromFileInternal`` there is whole path provided, so we do not need to care about this particular issue.

Still it would be better to extract it to common directory instead of content root, but that's debatable

Reply:
No, I meant the idea that user could install actual games via Install Content and they would show up in Xenia game list.

To clarify, Sonic 4: Episode 2 uses XamContentCreate to check if you have Episode 1 installed.

Reply:
Can you check it with Sonic? 
I tried to check it with TWAU, but to no avail. In main menu I installed second chapter and it log there is info that notification was received and it found it (reloaded DLCs), but it doesn't refresh it in-game. It only shows up after restarting game

Reply:
Installing XBLA package makes it go into shared content dir as expected. However, Sonic 4: Episode 2 can't find installed Episode 1. It calls XamContentAggregateCreateEnumerator with 0xFFFFFFFFFFFFFFFF xuid:
```
d> F80000D4 XamContentAggregateCreateEnumerator(FFFFFFFFFFFFFFFF, 00000000, 000D0000, 00000000, 706CFD14(00000000))
i> F80000D4 Added handle:F80000DC for class xe::kernel::XObject
F> F80000D4 StfsContainerDevice::ResolvePath(\Content\0000000000000000)
d> F80000D4 XamContentAggregateCreateEnumerator: added 0 items to enumerator
```